src/pages/patientView/structuralVairnt/PatientViewStructuralVariantTable.tsx

```diff
@@ -0,0 +1,403 @@
+        });
+
+        const GenePanelProps = (d: StructuralVariant[]) => ({
+            data: d.map(datum => ({
+                sampleId: datum.sampleId,
+                entrezGeneId: datum.site1EntrezGeneId,
+            })),
+            sampleToGenePanelId: this.props.store
+                .sampleToStructuralVariantGenePanelId.result!,
+            sampleManager: this.props.store.sampleManager.result!,
+            genePanelIdToGene: this.props.store.genePanelIdToEntrezGeneIds
+                .result!,
+            onSelectGenePanel: this.props.onSelectGenePanel,
+        });
+
+        columns.push({
+            name: 'Gene panel',
+            render: (d: StructuralVariant[]) =>
+                PanelColumnFormatter.renderFunction(GenePanelProps(d)),
+            download: (d: StructuralVariant[]) =>
+                PanelColumnFormatter.download(GenePanelProps(d)),
+            sortBy: (d: StructuralVariant[]) =>
+                PanelColumnFormatter.getGenePanelIds(GenePanelProps(d)),
+            visible: false,
+            order: 40,
+        });
+
+        columns.push({
+            name: 'Annotation',
+            render: (d: StructuralVariant[]) =>
+                AnnotationColumnFormatter.renderFunction(d, {
+                    uniqueSampleKeyToTumorType: this.props.store
+                        .uniqueSampleKeyToTumorType,
+                    oncoKbData: this.props.store.structuralVariantOncoKbData,
+                    oncoKbCancerGenes: this.props.store.oncoKbCancerGenes,
+                    usingPublicOncoKbInstance: this.props.store
+                        .usingPublicOncoKbInstance,
+                    enableOncoKb: AppConfig.serverConfig.show_oncokb as boolean,
+                    pubMedCache: this.props.store.pubMedCache,
+                    enableCivic: false,
+                    enableMyCancerGenome: false,
+                    enableHotspot: false,
+                    userEmailAddress: ServerConfigHelpers.getUserEmailAddress(),
+                    studyIdToStudy: this.props.store.studyIdToStudy.result,
+                }),
+            sortBy: (d: StructuralVariant[]) => {
+                return AnnotationColumnFormatter.sortValue(
+                    d,
+                    this.props.store.oncoKbCancerGenes,
+                    this.props.store.usingPublicOncoKbInstance,
+                    this.props.store.structuralVariantOncoKbData,
+                    this.props.store.uniqueSampleKeyToTumorType
+                );
+            },
+            order: 45,
+        });
+
+        columns.push({
+            name: 'Variant Class',
+            render: (d: StructuralVariant[]) => (
+                <span>{d[0].variantClass}</span>
+            ),
+            filter: (
+                d: StructuralVariant[],
+                filterString: string,
+                filterStringUpper: string
+            ) => {
+                return d[0].variantClass.indexOf(filterStringUpper) > -1;
```

Comment:
i know we do this in a lot of places already, but i dislike depending on case sensitive detection.     devs need to know that variantClass is always upper case.  instead we should just use RegExp(... , i).test

Reply:
yes or just do variantClass.toUpperCase().indexOf